### ISSUE TITLE
Other app language causes Bible link to show message "Please download Ge 1"

### ISSUE THREAD

**Issue by timbze** [AndBible/and-bible#2147]
https://support.andbible.org/scp/tickets.php?id=1396

**Describe the bug**
Link from commentary can not open when app language is set to German.

**Bug was found on And Bible version**
Build 643 standard release

**To Reproduce**
1. Set app language to German
2. Open Gen 1:18 in KJV window 1
3. Open Gill commentary for Gen 1:18 in window 2 (or anywhere)
4. Try to open the first link in the commentary (Ge 1:16)
5. Message says "Please download Ge 1"

It's important to note that many other links work, including "Ps 136:1/Ec 11:7" which is in the commentary further down from the same verse.

**Screenshots**
![image](https://user-images.githubusercontent.com/35117769/163680692-17fb23b0-a9cc-439a-88bf-667641053c78.png)

**Comment by tuomas2** [comment]
Hmm. OSIS produced by JSword is different depending on app language.

**Comment by tuomas2** [comment]
JSword: BibleNames.java:114 (getBook) finds with fuzzy search (line 128) something wrong for "Ge" in German language, and such a key does not exist in bible module. 

Not sure how to fix that. Perhaps should revert the order, such that english lookup is always first. But that could lead to other issues...

**Comment by timbze** [comment]
> Perhaps should revert the order, such that english lookup is always first.

I have not looked at the details of what you're working with, so I may be off... but would it work to make it that the first lookup language would be the module language?

**Comment by tuomas2** [comment]
You are right. We just should pass doc language information through quite many steps until it is used.

![Selection_428](https://user-images.githubusercontent.com/5811789/166878554-0d1c0329-69c8-4dd4-9882-3f8034316cb8.png)

**Comment by tuomas2** [comment]
Not very appealing thing to do from code point of view..
